tests/models/superpoint/test_modeling_superpoint.py

```diff
@@ -275,11 +275,13 @@ def test_inference(self):
         self.assertEqual(outputs.keypoints.shape, expected_keypoints_shape)
         self.assertEqual(outputs.scores.shape, expected_scores_shape)
         self.assertEqual(outputs.descriptors.shape, expected_descriptors_shape)
-        expected_keypoints_image0_values = torch.tensor([[480.0, 9.0], [494.0, 9.0], [489.0, 16.0]]).to(torch_device)
+        expected_keypoints_image0_values = torch.tensor([[0.75, 0.0188], [0.7719, 0.0188], [0.7641, 0.0333]]).to(
+            torch_device
+        )
         expected_scores_image0_values = torch.tensor(
-            [0.0064, 0.0137, 0.0589, 0.0723, 0.5166, 0.0174, 0.1515, 0.2054, 0.0334]
+            [0.0064, 0.0139, 0.0591, 0.0727, 0.5170, 0.0175, 0.1526, 0.2057, 0.0335]
         ).to(torch_device)
-        expected_descriptors_image0_value = torch.tensor(-0.1096).to(torch_device)
+        expected_descriptors_image0_value = torch.tensor(0.0449).to(torch_device)
```

Comment:
Wondering why scores and descriptors also changed, do you have an idea?

Reply:
Ok this is weird, I tried to backtrack where a change could happen.
I rolled back to the point I checked out commit 56baa03380fc17d85705240ebbc57c075a9b3f23 which is from the original SuperPoint PR that passed the tests. At least was passing the tests, but it turns out I end up with the same error at this revision.
I don't know if it's something hardware, maybe you can try on your side and see if the tests are different between the two commits ?
I'm not available tonight but can have a look tomorrow morning

Reply:
Noticed that the absolute to relative coordinate conversion was done too early and had an impact on the descriptor values. So I moved it to the end of the model and the descriptor value is back to a "closer" value than the initial one, yet it's still slightly different from the initial integration test